packages/near-fast-auth-signer/.gitignore

```diff
@@ -25,3 +25,6 @@ yarn-error.log*
 
```

Comment:
We probably need a re-design of this too 
<img width="427" alt="Screenshot 2024-02-06 at 9 08 23 PM" src="https://github.com/near/fast-auth-signer/assets/22358004/308ce490-7c52-47b7-8639-90b9a1939f99">